schema/cadence/schema.cql

```diff
@@ -115,6 +115,14 @@ CREATE TYPE request_cancel_info (
   cancel_request_id text,
 );
 
+-- External workflow signal in progress mutable state
+CREATE TYPE signal_info (
+  initiated_id      bigint,
+  signal_request_id text,
+  signal_name       text,
+  input             blob,
```

Comment:
Do you need to store control here too?  I would just store the entire signal initiated event like we do for workflowexecutionstartedevent.

Reply:
Added